src/scss/vendors/_vendors.scss

```diff
@@ -31,6 +31,45 @@
 	}
 }
 
+// uio constrast focus and hover
+@each $theme, $foregroundColor, $backgroundColor in $uio-themes {
+	.gpii-ext-theme-#{$theme},
+	.fl-theme-#{$theme} {
+		@include uio-focus-style($foregroundColor, $backgroundColor);
+
+		#defaultContainer button {
```

Comment:
Why do you add left and right padding to buttons on high contrast themes?

Reply:
I think I figured out. It's to make the hover/focus box not too close to the edge of button texts.

However, this usage seems only applied to UIO buttons. It would be better to use UIO buttton selectors instead of a generic "button" tag that may cause problems when other buttons are added to the website.